### ISSUE TITLE
Update io_bazel_rules_go to v0.10.3 [phab:D2247]

### ISSUE THREAD

**Issue by kythe** [google/kythe#2711]
_Created by **schroederc** at 2018-04-06 21:49:05_:

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2018-04-06 21:50:09_

**Comment by kythe** [comment]
_**schroederc** wrote at 2018-04-06 22:01:15_:

I pushed the review too soon.  There's an error now:

```
bazel-out/k8-fastbuild/bin/kythe/proto/linux_amd64_stripped/any_go_proto~/github.com/golang/protobuf/ptypes/any/any.pb.go:41:0: undefined: any.m·2
bazel-out/k8-fastbuild/bin/kythe/proto/linux_amd64_stripped/any_go_proto~/github.com/golang/protobuf/ptypes/any/any.pb.go:48:0: undefined: any.m·2
```

**Comment by kythe** [comment]
_**schroederc** wrote at 2018-04-06 22:12:01_:

Fix any_proto usage

**Comment by kythe** [comment]
_**schroederc** closed via commit b119bfabb60d636c86bd2618d170aa1b5a9e0adb at 2018-04-06 22:12:58_
